cla_common/laalaa.py

```diff
@@ -0,0 +1,38 @@
+import requests
+
+
+class LaaLaaError(Exception):
+    pass
+
+
+class LaalaaProviderCategoriesApiClient(object):
+    instance = None
+    def __init__(self, laalaa_base_url, category_translator):
+        self.categories = {}
+        self.category_translator = category_translator
+        self.laalaa_base_url = laalaa_base_url
+
+    def get_categories(self):
+        return self.categories or self._load_categories()
+
+    def _load_categories(self):
+        categories = self._fetch_categories()
+        # Translate categories and force keys to lower case
+        for category in categories:
+            code = category["code"].lower()
```

Comment:
🙋‍♂ We can ensure this translation happens in the output of LAALAA, so it doesn't mandate using shared client code.